dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/RepositoryImpl.java

```diff
@@ -2527,7 +2526,7 @@ else if (Slice.class.equals(multiType) || Page.class.equals(multiType) || pagina
                             // TODO remove doPriv once switched to Java 21 only or EclipseLink bug is fixed
```

Comment:
Sorry I didn't catch this earlier... but since this change resolves the "TODO" comment, please remove the comment that indicates the doPriv needs to be removed, as it will now be gone.

Reply:
Resolved